content/en/blog/_posts/2021-06-14-a-better-way-to-provision-kubernetes-resources-using-terraform.md

```diff
@@ -0,0 +1,339 @@
+---
+layout: blog
+title: 'A Better Way to Provision Kubernetes Resources Using Terraform'
+date: 2021-06-14
+slug: a-better-way-to-provision-kubernetes-resources-using-terraform
+---
+
+**Author:** Philipp Strube (Kubestack)
+
+After months of work and thousands of lines of Go, HCL and YAML I am very excited to share the release of the new [Kustomize based Terraform modules for Kubernetes](https://www.kubestack.com/catalog).
+The modules use the [Kustomization provider](https://registry.terraform.io/providers/kbst/kustomization/latest) I maintain as part of my work on the [Kubestack framework](https://www.kubestack.com/).
+Kubestack is an open-source Terraform framework and strives to deliver the best developer experience for teams working with Terraform and Kubernetes.
+
+I believe the new modules are a better way to provision Kubernetes resources using Terraform, because they bring three distinct benefits over the existing alternatives:

```

Comment:
```suggestion
I believe the new modules are a better way to provision Kubernetes resources using Terraform because they bring three distinct benefits over the existing alternatives:

```